tests/test_vis.py

```diff
@@ -463,6 +482,7 @@ def test_vegalite_heatmap_flag_config():
     df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/airbnb_nyc.csv")
     df = df.copy()
     assert not df.recommendation["Correlation"][0]._postbin
+    assert "Geographic" not in df.recommendation
```

Comment:
have you looked into Matplotlib supported Geographic charts?

Reply:
just wondering for future PRs:)

Reply:
Hi @caitlynachen! Yes (: Matplotlib has an API called [basemap](https://matplotlib.org/basemap/) which allows you to make geographic map projections just like the one I did in Altair. There's a good example of the proportional symbol map in [this article](https://jakevdp.github.io/PythonDataScienceHandbook/04.13-geographic-data-with-basemap.html). Let me know what you think!